numbers&methods.py

```diff
@@ -0,0 +1,18 @@
+monthly_budget = 15000
+
```

Comment:
class name should not consist of special characters and use any one naming convention